include/alpaka/mem/view/ViewPlainPtr.hpp

```diff
@@ -251,13 +245,13 @@ namespace alpaka
         };
 #endif
         //! The ViewPlainPtr offset get trait specialization.
-        template<typename TIdxIntegralConst, typename TDev, typename TElem, typename TDim, typename TIdx>
-        struct GetOffset<TIdxIntegralConst, ViewPlainPtr<TDev, TElem, TDim, TIdx>>
+        template<typename TDev, typename TElem, typename TDim, typename TIdx>
+        struct GetOffsets<ViewPlainPtr<TDev, TElem, TDim, TIdx>>
         {
             ALPAKA_FN_HOST
-            static auto getOffset(ViewPlainPtr<TDev, TElem, TDim, TIdx> const&) -> TIdx
+            auto operator()(ViewPlainPtr<TDev, TElem, TDim, TIdx> const&) const -> Vec<TDim, TIdx>
             {
-                return 0u;
+                return {};
```

Comment:
IMO should be `return {0}` to not rely on any default initialization. Same as in [ViewStdArray.hpp](https://github.com/alpaka-group/alpaka/pull/2080/files#diff-7a7202f988bb3302cc16ddcbead90f348b1107b34b83968c3749f0134158a806)

Reply:
But wouldn't this be a compile error for a zero-dimensional view? And `return {0}` would suggest that the returned vector is 1-dimensional. Even though `{0}` would also initialize any other excess dimensions with `0`. My version covers all dimension.

Reply:
I replaced it by `Vec<TDim, TIdx>::zeros()`.